### ISSUE TITLE
NGINX HA SSL Proxy Fails to run after upgrade to 3.1.2

### ISSUE THREAD

**Issue by reotto613** [home-assistant/addons#2550]
### Describe the issue you are experiencing

I upgraded to 3.1.2, and I am now seeing this in the log and I cannot access via my web url.

Log shows:

s6-overlay-suexec: fatal: can only run as pid 1

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

NGINX Home Assistant SSL proxy

### What is the version of the add-on?

3.1.2

### Steps to reproduce the issue

1. Update NGINX to 3.1.2
2. Fails to load, and site is no longer accessible from the web URL
3.
...

### Anything in the add-on logs that might be useful for us?

```txt
s6-overlay-suexec: fatal: can only run as pid 1
```

**Comment by Delta1977** [comment]
Solved in 3.1.4 https://github.com/home-assistant/addons/pull/2547

But 3.1.4 has other problem

**Comment by flinthamm** [comment]
v3.1.4 now fails here...
```
NGINX Home Assistant SSL proxy
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
parse error: Expected string key before ':' at line 1, column 4
[22:34:41] ERROR: Unknown HTTP error occured
parse error: Expected string key before ':' at line 1, column 4
[22:34:41] ERROR: Unknown HTTP error occured
[22:34:41] INFO: Running nginx...
nginx: [emerg] invalid port in upstream "homeassistant.local.hass.io:" in /etc/nginx.conf:62
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
```

Hope this helps.
